plugins/fireblocks/provider-gas-wrapper.ts

```diff
@@ -68,19 +68,103 @@ const polygonGasStation = async (
   };
 };
 
+/**
+ * {"status":"1","message":"OK","result":{"LastBlock":"15691953","SafeGasPrice":"8","ProposeGasPrice":"9","FastGasPrice":"9","suggestBaseFee":"7.376342631","gasUsedRatio":"0.9845104,0,0.978863833333333,0.2985338,0.4101417"}}
+ */
+
+interface EtherscanGasStationResponse {
+  SafeGasPrice: string;
+  ProposeGasPrice: string;
+  FastGasPrice: string;
+  suggestBaseFee: string;
+  gasUsedRatio: string;
+  LastBlock: number;
+}
+
+interface EthereumGasCache {
+  result: EtherscanGasStationResponse | undefined;
+  lastUpdated: number;
+}
+
+let ethereumGasCache: EthereumGasCache = {
+  result: undefined,
+  lastUpdated: 0,
+};
+
+const ethereumGasStation = async (
+  level: GasSpeed,
+  url: string
+): Promise<FeeData> => {
+  // Free tier of etherscan gas api has a 1req/5sec rate limit.
+  if (
+    ethereumGasCache.lastUpdated === undefined ||
+    ethereumGasCache.lastUpdated < Date.now() - 6000
+  ) {
+    const response = await fetch(url);
+    const { result } = await response.json();
+    ethereumGasCache = {
+      result,
+      lastUpdated: Date.now(),
+    };
+  }
+
+  const fees = ethereumGasCache.result;
+  if (fees === undefined) {
+    throw new Error(`Failed to load gas api data.`);
+  }
+  let feeForLevel: string;
+  switch (level) {
+    case 'safeLow': {
```

Comment:
Extension of above comment
```suggestion
    case GasSpeed.SAFE_LOW: {
```